samples/tab-request-approval/csharp/TabRequestApproval/Controllers/HomeController.cs

```diff
@@ -118,53 +117,45 @@ public async Task<ActionResult> SendNotificationToManager(RequestInfo taskInfo)
                     var graphClient = SimpleGraphClient.GetGraphClient(taskInfo.access_token);
 
                     // Retrieve user information from Microsoft Graph API
-                    var user = await graphClient.Users[taskInfo.personaName]
-                              .Request()
-                              .GetAsync();
+                    var user = await graphClient.Users[taskInfo.personaName].GetAsync();
 
                     // Retrieve installed apps for the user from Microsoft Graph API
-                    var installedApps = await graphClient.Users[user.Id].Teamwork.InstalledApps
-                                       .Request()
-                                       .Expand("teamsApp")
-                                       .GetAsync();
+                    var installedApps = await graphClient.Users[user.UserPrincipalName].Teamwork.InstalledApps
+                                       .GetAsync((requestConfiguration) =>
+                                       {
+                                           requestConfiguration.QueryParameters.Expand = new string[] { "teamsAppDefinition" };
+                                       });
 
                     // Filter installed apps to find the one with DisplayName "Tab Request Approval"
-                    var installationId = installedApps.Where(id => id.TeamsApp.DisplayName == "Tab Request Approval").Select(x => x.TeamsApp.Id);
+                    var installationId = installedApps.Value.Where(id => id.TeamsAppDefinition.DisplayName == "Tab Request Approval").Select(x => x.TeamsAppDefinition.Id);
```

Comment:
@pankajmunde12  rest changes working fine accept  instead of 
 var installationId = installedApps.Value.Where(id => id.TeamsAppDefinition.DisplayName == "Tab Request Approval").Select(x => x.TeamsAppDefinition.Id);
It should be 
 var installationId = installedApps.Value.Where(id => id.TeamsAppDefinition.DisplayName == "Tab Request Approval").Select(x => x.TeamsAppDefinition.TeamsAppId);

Reply:
Thanks @Pawank-MSFT !
I have updated the code. Please check.